docs/operations/common_configuration/overview.rst

```diff
@@ -0,0 +1,9 @@
+.. _common_configuration_overview:
+
+Common configuration overview
+=============================
```

Comment:
I suggest renaming this file to How Tos/FAQs as this ties in nicely with #568 . This can then be a place where we can pile config snippets for various functions (zone aware routing, tracing, circuit breakers, etc.).